docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: model
+title: Pre-trained Pipeline for Few-NERD-General NER Model
```

Comment:
`Pretrained pipeline for the Few-NERD-General model`

Reply:
Preventing technical details is for demos. In ModelsHub pages we should also be careful, but I think this is ok.

Reply:
`Pretrained`, without hyphen. Prefixes are not hyphenated by  default:
![image](https://user-images.githubusercontent.com/36634572/144821608-93d20f60-89e7-498d-b0c2-461bafe7d7a1.png)